test_scripts/smoke_api.lua

```diff
@@ -514,7 +514,7 @@ end
 
 -- 2. PutFiles
 	commonSteps:PutFile("PutFile_action.png", "action.png")
-	commonSteps:PutFile("PutFile_MaxLength_255Characters", strMaxLengthFileName255)
+	commonSteps:PutFile("PutFile_MaxLength_255Characters", strMaxLengthFileName242)
```

Comment:
`PutFile_MaxLength_255Characters` to `PutFile_MaxLength_242Characters `

Reply:
@jacobkeeler Change won't be visible after re-base on develop

Reply:
@jacobkeeler Branch is re-based on current develop